bot/code_review_bot/tasks/clang_tidy_external.py

```diff
@@ -127,6 +130,13 @@ def as_markdown(self):
             ),
         )
 
+    def is_publishable(self):
+        """
+        Is this issue publishable on reporters ?
+        """
+
+        return super().is_publishable and not self.check.startswith("clang-diagnostic")
```

Comment:
What about doing `self.check.startswith("mozilla-civet-")` instead of `not self.check.startswith("clang-diagnostic")`?